BUILD.gn

```diff
@@ -1404,7 +1404,8 @@ dist_zip("hunspell_dictionaries_zip") {
 }
 
 copy("libcxx_headers") {
-  sources = libcxx_headers + libcxx_licenses
+  sources = libcxx_headers + libcxx_licenses +
+            [ "//buildtools/third_party/libc++/__config_site" ]
```

Comment:
Commented in https://github.com/electron/electron-rebuild/pull/703/files#r660348710 to make sure downstream is aware of the change.